src/protocol/V1.php

```diff
@@ -41,10 +41,11 @@ public function init(...$args): array
             throw new MessageException($output['message'] . ' (' . $output['code'] . ')');
         }
 
-        return $signature == self::SUCCESS ? $output : [];
+        return $output;
```

Comment:
Init has server response only success or failure. Failure throws exception and that means it's always success otherwise.